.circleci/config.yml

```diff
@@ -52,12 +52,13 @@ jobs:
       - run:
           name: Start server with correct env for cypress tests
           command: yarn cypress:dev
-          wait-on: https://localhost:3000
           background: true
```

Comment:
the original `wait-on` parameter wasn't doing anything apparently as parameters are used for circleci commands if I understand correctly, but this is just a direct invocation of a bash command (https://circleci.com/docs/2.0/reusing-config/#using-the-parameters-declaration)